src/cloudai/systems/slurm/strategy/slurm_command_gen_strategy.py

```diff
@@ -175,20 +176,48 @@ def _write_sbatch_script(self, args: Dict[str, Any], env_vars_str: str, srun_com
             f"#SBATCH -N {args['num_nodes']}",
         ]
 
+        self._append_sbatch_directives(batch_script_content, args, output_path)
+
+        batch_script_content.append(env_vars_str)
```

Comment:
I liked empty lines as separators between these blocks, IMO it improves readability.

Reply:
Didn't quite follow this comment.

Reply:
Sorry, let me clarify. Before the change, it was like this: `batch_script_content.extend(["", env_vars_str, "", srun_command])`. This adds two empty lines between logical blocks (sbatch and env vars, env vars and srun args). IMO it contributed to readability of the final sbatch script.

Reply:
Done